models/product-type/src/product-type-draft/presets/sample-data-fashion/accessories.ts

```diff
@@ -0,0 +1,90 @@
+import { AttributeDefinitionDraft } from '@commercetools-test-data/attribute-definition';
+import * as AttributeType from '@commercetools-test-data/attribute-type';
+import { LocalizedString } from '@commercetools-test-data/commons';
+import * as ProductTypeDraft from '../..';
+import { TProductTypeDraftBuilder } from '../../../types';
+
+const accessories = (): TProductTypeDraftBuilder =>
+  ProductTypeDraft.presets
+    .empty()
+    .key('accessories')
+    .name('Accessories')
+    .description('Accessories')
+    .attributes([
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('type')
+        .label(LocalizedString.presets.empty()['en-US']('Type'))
+        .inputTip(LocalizedString.presets.empty()['en-US']('Type'))
+        .isRequired(true)
+        .type(
+          AttributeType.presets.enumType().values([
+            {
+              key: 'Jewelry',
+              label: 'Jewelry',
+            },
+            {
+              key: 'Bag',
+              label: 'Bag',
+            },
+            {
+              key: 'Shoes',
+              label: 'Shoes',
+            },
+          ])
+        )
+        .attributeConstraint('None')
+        .isSearchable(true)
+        .inputHint('SingleLine'),
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('engraving')
+        .label(LocalizedString.presets.empty()['en-US']('Engraving'))
+        .inputTip(LocalizedString.presets.empty()['en-US']('Engraving'))
+        .isRequired(false)
+        // @ts-ignore
+        .type(AttributeType.random().name('text'))
+        .attributeConstraint('None')
+        .isSearchable(false)
+        .inputHint('SingleLine'),
+    ])
+    .attributeDefinitions([
```

Comment:
The default transformer (currently) builds both:
```
const transformers = {
  default: Transformer<TProductTypeDraft, TProductTypeDraft>('default', {
    buildFields: ['attributes', 'attributeDefinitions'],
  }),
```
Seems like the default should remove `attributeDefinitions`, but I'm not sure.

Reply:
My vote is that it should only build `attributes` and `attributeDefinitions` is constructed in the GraphQL transformer. @jaikamat does this align with our current pattern?

Reply:
REST representation:
![image](https://user-images.githubusercontent.com/15024562/232569119-52042477-79de-4c70-bc97-fe898a205ed3.png)

GraphQL representation:
![image](https://user-images.githubusercontent.com/15024562/232569198-5272b1ba-766d-4791-9b57-a35e985ba77d.png)

Since both use `AttributeDefinitionDraft[]` It does seem like consumers should only be adding `attributes` and we can use the GraphQL transformer transformer to specify `attributeDefinition` instead

Reply:
As for whether this aligns with our current pattern, I believe it does. There were exceptions made for a particular edge case where the graphql transforms had properties that could not be inferred from the REST type (the model escapes me at the moment) and we had to apply both properties for consumers to use, but this doesn't appear to be one of these cases

Reply:
Thank you both! updating...